### PR TITLE
Ansys em19.0.documentation

### PR DESCRIPTION
**Pull request by me1bdh**

### CODE REVIEW

File: sharc/software/apps/ansysem.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@ Ansys EM can be activated using the module files::
     module load apps/ansysem/17.2
     module load apps/ansysem/18.0/binary
     module load apps/ansysem/18.2/binary
+	module load apps/ansysem/19.0/binary
```
Comment (willfurnass) [inline sharc/software/apps/ansysem.rst]: Why the indentation?

Comment (me1bdh) [reply sharc/software/apps/ansysem.rst]: David asked this yesterday. I have not purposely indented this. It isn't present locally on after I push. Maybe something with Travis?

File: sharc/software/apps/ansysem.rst [hunk 2]
```diff
@@ -21,6 +21,7 @@ Ansys EM can be activated using the module files::
     module load apps/ansysem/17.2
     module load apps/ansysem/18.0/binary
     module load apps/ansysem/18.2/binary
+    module load apps/ansysem/19.0/binary
```
Comment (me1bdh) [inline sharc/software/apps/ansysem.rst]: Indentation removed. Looking on my local machine to find out why this is happening (only occurs on new edits)

Comment (willfurnass) [reply sharc/software/apps/ansysem.rst]: Suspect it's got something to do with the file containing a mix of spaces vs tabs.

**Pull request merged**